docs/source/notebooks/SMC2_gaussians.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<ul><li>So far we have <strong>seen</strong> that the SMC sampler is just ...</li><li>SMC proceed<strong>s</strong> by moving <em>sequentially</em> through a series of stages, starting from a<strong>n</strong> easy to sample distribution until it get<strong>s</strong> to the posterior distribution. </li><li>The idea is as follow<strong>s:</strong> the...</li><li>SMC starts with β=0 and progress<strong>es</strong> by always</li><li>As β increase<strong>s,</strong> the tempered posterior concetrates more and more. This is represented in the <s>avobe</s> <strong>above</strong> figure</li><li>"For the SMC version implemented in PyMC3 the number of parallel chains (or individuals) is the number of draws": to be sure, this means that, at each stage, SMC runs as many MAakov chains as the number of draws?</li><li>This can <strong>be</strong> adjusted if necessary by the <code>threshold</code> parameter (in the interval [0, 1]) <strong>-- the current default of 0.5 is generally considered as a good default.</strong> The larger this value<strong>,</strong> the higher the target ESS and the closer <s>the smaller the differences</s> two succes<strong>s</strong>ive values of β will be. This ESS value<strong>s are</strong> <s>is</s> computed from the importance weights and not from autocorrelations as ArviZ does (for example using <code>az.ess</code> or <code>az.summary</code>).</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3981/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>